### DISCUSSION TITLE
Suricata broken post update

### DISCUSSION THREAD

**Discussion by WPTechnician** [Security-Onion-Solutions/securityonion#2815] [Unsupported Versions] [answered]
Hi:

Since upgrading to 2.3.21 I am only getting Zeek in my logs.  Checking the suricata logs on my sensor I get the following:

"WARNING: Error loading config file: .dockercfg: $HOME is not defined"

Is anyone else having this issue or know of a way to get it resolved?  Thanks!

**Comment by WPTechnician** [comment, marked as answer]
The warning is still throwing but it seems to be working now after a rebooting everything.

**Comment by TOoSmOotH** [comment]
That warning is a known issue and should not be a problem.
